torch/csrc/jit/passes/quantization.cpp

```diff
@@ -182,12 +182,13 @@ class ObserveHelper {
       Graph* g,
       script::Module& module,
       const QConfig& qconfig);
-  void propagateWeightValues(Node* n, std::shared_ptr<Graph>& graph);
+  void propagateValues(Node* n, std::shared_ptr<Graph>& graph);
```

Comment:
This name became very generic - please add a comment what exactly is getting propagated and where.